src/pages/[platform]/build-a-backend/storage/copy/index.mdx

```diff
@@ -46,16 +46,40 @@ import flutter0 from '/src/fragments/lib/storage/flutter/copy.mdx';
 
 <InlineFilter filters={['javascript', 'react-native', 'angular', 'nextjs', 'react', 'vue']}>
 
-`copy` method copies an existing file to a different key and returns a `{key: 'destKey'}` object upon success.
+The `copy` API duplicates an existing file to a designated path and returns an object `{path: 'destPath'}` upon successful completion.
 
 <Callout>
 
 `copy` can copy an object up to 5 GB in a single operation.
 
 </Callout>
 
-To copy a file, you need to provide a source key in `source` and a destination key in `destination`. `accessLevel` can be 'guest', 'private', or 'protected', defaulting to 'guest'.
+To copy a file, you need to provide the existing path in `source` and the destination path in `destination`.
 
+<BlockSwitcher>
+<Block name="With Path (Above v6.2.0)">
+```javascript
+import { copy } from 'aws-amplify/storage';
+
+const copyFile = async () => {
+  try {
+    const response = await copy({
+      source: {
+        path: 'public/album/2024/1.jpg',
+        // Alternatively, path: ({identityId}) => `protected/{identityId}/album/1.jpg`
+      },
+      destination: {
+        path: 'public/shared/2024/1.jpg',
+        // Alternatively, path: ({identityId}) => `protected/{identityId}/shared/1.jpg`
```

Comment:
Don't think this is going to work without the `$`, is it?
```suggestion
        // Alternatively, path: ({identityId}) => `protected/${identityId}/shared/1.jpg`
```

Reply:
Thanks scott for the catch fixed it :)